app/src/main/java/com/android/sample/ui/friends/NewConnectionsContent.kt

```diff
@@ -45,8 +46,8 @@ fun NewConnectionsContent(
           onResult = { results -> searchResults.value = results },
           onFailure = { exception -> Log.e(logTag, "Search failed", exception) })
     } else {
-      searchResults.value = sentRequests
-      Log.d(logTag, " Length: ${sentRequests.size}")
+      searchResults.value = emptyList()
+      Log.d(logTag, "Search query is blank")
```

Comment:
Here you have a good use of logtag , can you do the same for my previous comment related to log output ?